textattack/models/helpers/utils.py

```diff
@@ -10,17 +10,14 @@
 
 import torch
 
-from textattack.shared import utils
-
 
 def load_cached_state_dict(model_folder_path):
-    model_folder_path = utils.download_if_needed(model_folder_path)
     # Take the first model matching the pattern *model.bin.
     model_path_list = glob.glob(os.path.join(model_folder_path, "*model.bin"))
     if not model_path_list:
         raise FileNotFoundError(
             f"model.bin not found in model folder {model_folder_path}."
         )
     model_path = model_path_list[0]
-    state_dict = torch.load(model_path, map_location=utils.device)
+    state_dict = torch.load(model_path, map_location=torch.device("cpu"))
```

Comment:
Just to clarify, the reason we do this, is to always load params on the CPU and then transfer to GPU as when required?

Reply:
Yes